packages/odyssey-react-mui/src/labs/DataFilters.tsx

```diff
@@ -604,6 +604,7 @@ const DataFilters = ({
                               <AutocompleteInnerContainer>
                                 <Autocomplete
                                   hasMultipleChoices
+                                  isCustomValueAllowed
```

Comment:
@KevinGhadyani-Okta Do you think we want to just allow this all the time within data filters?

Reply:
I want more information on what this does.

Reply:
For posterity, we chatted about this on Slack but if anyone else comes across this, this prop allows the user to enter any value into the autocomplete. They are not required to select an option from the drop down.